raulbolsantos/get_mp3
# fixes #8 - Using Mp3Fiber service

This fix needs to be tested with different inputs.
Steps to test:
1 - Create a *musics.txt* file on the path *get_mp3/get_mp3/'*
2 - Add some music names in the file (one per line)
3 - Run *python get_mp3.py* 
4 - After the process complete, check if all the musics were successfully downloaded
